src/components/Button.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+const Button =  function(props){
+    // console.log(props)
+    return <button onClick={props.onClick}className="btn btn-primary fn-submit-name">{props.text}</button>
```

Comment:
I recommend using `Format Document` in VSCode as you can see missing space here can break the code